src/main/electron/listener.cljs

```diff
@@ -197,12 +197,14 @@
                              (editor-handler/insert (str "\n" content)))
 
                            (do
+                             (editor-handler/escape-editing)
                              (when (not= page (state/get-current-page))
                                (page-handler/create! page {:redirect? redirect-page?}))
-                             (editor-handler/escape-editing)
-                             (editor-handler/api-insert-new-block! content {:page page
-                                                                            :edit-block? true
-                                                                            :replace-empty-target? true}))))))
+                             ;; Or else this will clear the newly inserted content
+                             (js/setTimeout #(editor-handler/api-insert-new-block! content {:page page
```

Comment:
This is a temporary fix.
The original bug is at `editor-handler/api-insert-new-block!`:

When editing another block, `editor-handler/api-insert-new-block!` might insert content to both the current and newly created block.